### PR TITLE
Fix null mimetype issue on blobUpload

### PR DESCRIPTION
**Pull request by fherbin**
fix this issue https://github.com/nuxeo/nuxeo-php-client/issues/9
please release, it's a major issue.

### CODE REVIEW

File: src/Nuxeo/Client/Request.php [hunk 1]
```diff
@@ -79,13 +79,15 @@ public function addRelatedFile($file, $contentType = null) {
     $new = clone $this;
 
     try {
+      $headers = ['Content-Disposition' => $file->getContentDisposition()];
+      $contentType = $contentType ?? $file->getContentType();
+      if ($contentType) {
```
Comment (fherbin) [inline src/Nuxeo/Client/Request.php]: to avoid sending a header with a null value

File: src/Nuxeo/Client/Request.php [hunk 2]
```diff
@@ -79,13 +79,15 @@ public function addRelatedFile($file, $contentType = null) {
     $new = clone $this;
 
     try {
+      $headers = ['Content-Disposition' => $file->getContentDisposition()];
+      $contentType = $contentType ?? $file->getContentType();
```
Comment (fherbin) [inline src/Nuxeo/Client/Request.php]: Mr bug here

**Comment by Ludoutt** [comment]
please accept this PR @pgmillon, it's a major issue.
We must fix our "vendor" directory manually...

**Comment by pgmillon** [comment]
Hi,
I'm very sorry I couldn't get a chance to work on this project since a while.
Thank you very much for your interest and your contribution to the project.
It would be great to add a test to avoid this issue from appearing again in the future.
Probably an additional assertStringMatchesFormatFile in OperationTest.php: https://github.com/nuxeo/nuxeo-php-client/blob/fc2de05b49188752e2a10d15887a93195f111e94/tests/Nuxeo/Client/Tests/OperationTest.php#L164-L165

**Comment by pgmillon** [comment]
I spent some time today but could not reproduce any issue. I mean, there's no content-type header for the upload indeed but that's not a problem for Nuxeo to handle it.
Could you please look at my [comment](https://github.com/nuxeo/nuxeo-php-client/issues/9#issuecomment-777669216) and precise versions + share a code excerpt ?
Thanks

**Pull request closed without merging**